scalaz-test/src/main/scala/ztest/property.scala

```diff
@@ -0,0 +1,21 @@
+package scalaz.test
+
+import scalaz.{ Applicative, Traversable }
+import scalaz.Scalaz._
+
+object property {
+
+  def exhaustive[F[_]: Applicative, T[_]: Traversable, I, O](in: T[(() => I)])(testGenerator: I => F[O]): F[T[O]] =
```

Comment:
A comment on what this does and why it's called `exhaustive` would be nice.

Reply:
Sure.